### PR TITLE
Updated changelog/package.json for 4.3.3 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.3.3 (2020-09-11)
+
+- Fixed the tabset component to allow activating a tab as soon as it is created. [#132](https://github.com/blackbaud/skyux-tabs/pull/132)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: How about "Fixed the tabset component to activate tabs as soon as they're created."  ?

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.3.3](https://host.nxt.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.3.3)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md:3]: That changes the meaning a bit. The problem we're fixing is _allowing_ consumers to activate a new tab when it is created. We're not activating tabs when they're created; the problem only comes when a consumer wishes to activate a new tab immediately after being created.

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md:3]: Gotcha. Can we include that then? 

Fixed the tabset component to allow consumers to activate a tab as soon as they create it.

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Done.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.3.3](https://host.nxt.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.3.3)

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Pull request merged**